### ISSUE TITLE
Add ARIA attribute to element that opens the Tooltip component

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#2120]
# Description

In the [Tooltip component](http://design.if-me.org/?path=/docs/components-tooltip--element-is-text) ([code](https://github.com/ifmeorg/ifme/blob/main/client/app/components/Tooltip/index.jsx)), the element that opens the tooltip has an `aria-labelledby` attribute on it so that it can be associated with the element with `role="tooltip"`.  This element that opens the tooltip does not have a `role` itself. This will reduce the quality of the screen-reader experience.

This is a problem because the `aria-labelledby` attribute cannot be used on a `div` with no valid role attribute.

We don't want to assign a role to this `div` because it could contain an element of a different role. The ideal solution would be to add `aria-labelledby={tooltipId}` to the child that is rendered i.e. `{Utils.renderContent(element)}`.

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!

**Comment by PaxtonDevs** [comment]
Hello, I would like to assign myself this issue

**Comment by julianguyen** [comment]
@PaxtonDevs Added you to our org! After accepting the invitation, you should be able to assign yourself to the issue.

**Comment by PaxtonDevs** [comment]
Awesome, thank you!

**Comment by andomain** [comment]
Hi, if there is no further activity on this I'd like to contribute?

**Comment by a-camarillo** [comment]
Hi, I noticed this issue is still unresolved and would like to tackle it!

I took a look at [#2156] and suggest possibly breaking this issue up into two smaller issues depending on if we want to write custom functionality for rendering HTML as a React Element or use one of the [other libraries](https://github.com/hatashiro/react-render-html#deprecation) recommended by the old react-render-html maintainers.
